core/src/main/scala/com/criteo/cuttle/ExecutionContexts.scala

```diff
@@ -0,0 +1,77 @@
+package com.criteo.cuttle
+
+import java.util.concurrent.Executors
+
+import scala.concurrent.ExecutionContext
+import scala.language.implicitConversions
+import scala.util.Try
+
+object ExecutionContexts {
+
+  sealed trait WrappedExecutionContext {
+    val underlying: ExecutionContext
+  }
+
+  sealed trait Metrics {
```

Comment:
what is this for? to count the total threads created during the lifetime? A little comment can be helpful.

Reply:
Yes it is, to be renamed as threadPoolSize then